strings/ic_tips.txt

```diff
@@ -0,0 +1,28 @@
+To crack the safe in the vault, use a stethoscope or three plastic explosives on it.
+Interacting with a windoor rather then bumping into it will keep it open, you can interact with it again to close it.
+You can spray a fire extinguisher, throw items or fire a gun while floating through space to change your direction. Simply fire opposite to where you want to go.
+Firesuits and winter coats offer mild protection from the cold, allowing you to spend longer periods of time near breaches and space than if wearing nothing at all.
```

Comment:
I don't know if this is actually true for firesuits, those give heat protection. Drinking coffee gives cold resist however